Project.toml

```diff
@@ -7,6 +7,7 @@ version = "4.27.0"
 BandedMatrices = "aae01518-5342-5314-be14-df237901396f"
 BlockBandedMatrices = "ffab5731-97b5-5995-9138-79e8c1846df0"
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```

Comment:
Oof. Wouldn't this be circular dependency? Why do you need DifferentialEquations?